extensions/git/package.json

```diff
@@ -2556,8 +2566,12 @@
       ],
       "git.worktrees": [
         {
-          "command": "git.deleteWorktree",
+          "command": "git.createWorktree",
```

Comment:
This command should be limited with the `when` clause to `scmProviderContext == repository`